### DISCUSSION TITLE
Ring Steplight Battery?

### DISCUSSION THREAD

**Discussion by INVICTUS0913** [dgreif/ring#675] [Q&A] [answered]
Hello everyone, 
Does this work with Ring Steplight Battery? 
If it does, do I still need a Ring Brdige?
Thank you!

**Comment by dgreif** [comment, marked as answer]
Yep, it should work with all ring smart lighting. You do still need a lighting bridge as that is how the light gets connected to the ring servers, which is how the plugin communicates with the lights.

**Reply by INVICTUS0913** [reply]
Thanks! 
If it is possible, please update the step light into supported list in original instruction,  "Ring Smart Lights (Motion Detector, Flood/Path/Spot Lights, Transformer, Outdoor Smart Plug)"
